documentation/modules/managing/con-broker-config-properties.adoc

```diff
@@ -200,131 +201,142 @@ log.roll.ms=604800000
-
-For time-based log retention, you set a retention period based on hours, minutes and milliseconds.
-The retention period is based on the time messages were appended to the segment.
-
-The milliseconds configuration has priority over minutes, which has priority over hours. The minutes and milliseconds configuration is null by default, but the three options provide a substantial level of control over the data you wish to retain. Preference should be given to the milliseconds configuration, as it is the only one of the three properties that is dynamically updateable.
+In Kafka, log cleanup policies determine how log data is managed.
+In most cases, you won't need to change the default configuration at the cluster level, which specifies the `delete` cleanup policy and enables the log cleaner used by the `compact` cleanup policy:
 
 [source,env]
 ----
 # ...
-log.retention.ms=1680000
+log.cleanup.policy=delete
+log.cleaner.enable=true
 # ...
 ----
 
-If  `log.retention.ms` is set to -1, no time limit is applied to log retention, so all logs are retained.
-Disk usage should always be monitored, but the -1 setting is not generally recommended as it can lead to issues with full disks, which can be hard to rectify.
+Delete cleanup policy:: Delete policy is the default cluster-wide policy for all topics.
+The policy is applied to topics that do not have a specific topic-level policy configured. 
+Kafka removes older segments based on time-based or size-based log retention limits. 
+Compact cleanup policy:: Compact policy is generally configured as a topic-level policy (`cleanup.policy=compact`).
+Kafka's log cleaner applies compaction on specific topics, retaining only the most recent value for a key in the topic.
+You can also configure topics to use both policies (`cleanup.policy=compact,delete`). 
+
+.Setting up retention limits for the delete policy
+
+The delete policy corresponds to managing logs with data retention policies. 
+The policy is suitable when data does not need to be retained forever.
+You can establish time-based or size-based log retention and cleanup policies to keep logs bounded. 
 
-For size-based log retention, you set a maximum log size (of all segments in the log) in bytes:
+When log retention policies are employed, non-active log segments are removed when retention limits are reached.
+This deletion of old segments helps prevent exceeding disk capacity.
+
+For time-based log retention, you set a retention period based on hours, minutes, or milliseconds: 
 
 [source,env]
 ----
 # ...
-log.retention.bytes=1073741824
+log.retention.ms=1680000
 # ...
 ----
 
-In other words, a log will typically have approximately _log.retention.bytes/log.segment.bytes_ segments once it reaches a steady state.
-When the maximum log size is reached, older segments are removed.
+The retention period is based on the time messages were appended to the segment. The milliseconds configuration has priority over minutes, which has priority over hours. 
```

Comment:
We can make it clear that Kafka is taking _the largest timestamp message in one segment_ as the time to determine if this segment is expired or not.

Reply:
Added: _Kafka uses the timestamp of the latest message within a segment to determine if that segment has expired or not._